cmd/buildah/from.go

```diff
@@ -18,54 +18,59 @@ var (
 	fromFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "name",
-			Usage: "set a name for the working container",
+			Usage: "set a `name` for the working container",
 		},
 		cli.StringFlag{
 			Name:  "image",
-			Usage: "name of the starting image",
+			Usage: "name of the starting `image`",
 		},
-		cli.BoolFlag{
+		cli.BoolTFlag{
 			Name:  "pull",
 			Usage: "pull the image if not present",
 		},
 		cli.BoolFlag{
 			Name:  "pull-always",
-			Usage: "pull the image, even if a version is present",
+			Usage: "pull the image even if one with the same name is already present",
 		},
 		cli.StringFlag{
 			Name:  "registry",
-			Usage: "prefix to prepend to the image name in order to pull the image",
+			Usage: "`prefix` to prepend to the image name in order to pull the image",
 			Value: DefaultRegistry,
 		},
 		cli.StringFlag{
 			Name:  "signature-policy",
-			Usage: "signature policy path",
+			Usage: "`pathname` of signature policy file (not usually used)",
 		},
 		cli.BoolFlag{
```

Comment:
IDK, but thought I'd throw it by you.  Should the mount be using a BoolTflag also?